src/components/graph/menus/network-modification-node-editor.js

```diff
@@ -41,7 +41,8 @@ import LoadCreationDialog from '../../refactor/dialogs/load-creation/load-creati
 import GeneratorCreationDialog from '../../dialogs/generator-creation-dialog';
 import ShuntCompensatorCreationDialog from '../../dialogs/shunt-compensator-creation-dialog';
 import LineCreationDialog from '../../dialogs/line-creation-dialog';
-import TwoWindingsTransformerCreationDialog from '../../dialogs/two-windings-transformer/two-windings-transformer-creation-dialog';
+import { default as TwoWindingsTransformerCreationDialogOld } from '../../dialogs/two-windings-transformer/two-windings-transformer-creation-dialog';
+import TwoWindingsTransformerCreationDialog from '../../refactor/dialogs/two-windings-transformer-creation/two-windings-transformer-creation-dialog';
```

Comment:
you added the jsconfig.json you can use the absolute imports: 
`import TwoWindingsTransformerCreationDialog from 'components/refactor/dialogs/two-windings-transformer-creation/two-windings-transformer-creation-dialog';`